Week9/geg222/Mission2/src/components/CartList.tsx

```diff
@@ -0,0 +1,18 @@
+import CartItem from "./CartItem";
+import { useSelector } from "../hooks/useCustomRedux";
+
+const CartList = () => {
+  const { cartItems } = ({} = useSelector((state) => state.cart));
```

Comment:
현재 CartList에서 useSelector를 통해 상태를 가져올 때 중간에 불필요한 할당 연산과 빈 객체가 포함된 구조분해할당을 사용하고 있는데, 아래처럼 수정하면 코드 의도가 명확해지고 가독성도 올라갈 것 같습니다!
(참고 : https://eslint.org/docs/latest/rules/no-empty-pattern)
```suggestion
  const { cartItems } = useSelector((state) => state.cart);
```